src/views/AddPriceSingle.vue

```diff
@@ -30,7 +30,7 @@
             </h3>
             <v-sheet v-if="productMode === 'barcode'">
               <v-btn class="mb-2 mr-2" size="small" prepend-icon="mdi-barcode-scan" @click="showBarcodeScanner">{{ $t('AddPriceSingle.ProductInfo.ScanBarcode') }}</v-btn>
-              <a href="#" @click.prevent="showBarcodeManualInput">{{ $t('AddPriceSingle.ProductInfo.TypeBarcode') }}</a>
+              <v-btn class="mb-2 mr-2" size="small" prepend-icon="mdi-numeric" @click.prevent="showBarcodeManualInput">{{ $t('AddPriceSingle.ProductInfo.TypeBarcode') }}</v-btn>
```

Comment:
I'm wondering if we should have a different behavior on small screens : a shortened version to avoid having 2 lines. Can be done in a different PR of course

![image](https://github.com/openfoodfacts/open-prices-frontend/assets/7147385/92559cd8-117c-4ac2-8440-5b8fd7cc00ad)

Reply:
Was about to work on this! But no worries, great that you are helping @ArturLange , greatly needed.
I would shorten also to "Scan" and "Manual" (or similar). I was thinking of just using obvious icons too, like a "camera" icon and a "keypad" icon. Some UI/UX expert or designer input would be great

Reply:
The UX "good practice" (at least used at OFF) is to have always icon + text.

I'm adding a commit (and screenshot) to reduce the text on small screens.